aws_lambda_powertools/shared/cookies.py

```diff
@@ -0,0 +1,75 @@
+from datetime import datetime
+from enum import Enum
+from io import StringIO
+from typing import List, Optional
+
+
+class SameSite(Enum):
+    DEFAULT_MODE = ""
+    LAX_MODE = "Lax"
+    STRICT_MODE = "Strict"
+    NONE_MODE = "None"
+
+
+def _getdate(timestamp: datetime) -> str:
+    return timestamp.strftime("%a, %d %b %Y %H:%M:%S GMT")
+
+
+class Cookie:
+    def __init__(
+        self,
+        name: str,
+        value: str,
+        path: Optional[str] = None,
+        domain: Optional[str] = None,
```

Comment:
```suggestion
        path: str = "",
        domain: str = "",
```

Setting as an empty string will simplify your conditional branch (`if path and len(path) > 0`) to `if path`.

`path: Optional[str] = None` is useful in scenarios when an empty string is valid too - `if path is not None` would accept a `""`.